### DISCUSSION TITLE
Will desktop app be fully open source?

### DISCUSSION THREAD

**Discussion by trymeouteh** [hoppscotch/hoppscotch#3991] [Q&A] [answered]
I realized the builds of the desktop app are in this repo with no source code available.

https://github.com/hoppscotch/releases

When the app if officially released, will it be fully open source?

**Comment by liyasthomas** [comment, marked as answer]
Source code for desktop app: https://github.com/hoppscotch/hoppscotch/tree/main/packages/hoppscotch-selfhost-desktop
